build.gradle

```diff
@@ -197,6 +197,7 @@ dependencies {
     compile group: 'com.sun.mail', name: 'mailapi', version: '1.6.1'
     compile group: 'commons-lang', name: 'commons-lang', version: '2.6'
     compile group: 'commons-validator', name: 'commons-validator', version: '1.6'
+    compile group: 'com.jayway.jsonpath', name: 'json-path', version: '2.4.0'
```

Comment:
introduced jayway jsonpath to support jsonpath parser for Jackson.

Reply:
👍